api/src/main/java/org/openmrs/module/appointmentscheduling/api/db/hibernate/HibernateAppointmentDAO.java

```diff
@@ -251,4 +256,55 @@ private Criteria createAppointmentsInTimeSlotByStatusCriteria(
 		criteria.add(Restrictions.in("status", statuses));
 		return criteria;
 	}
+
+	@Override
+	public List<AppointmentDailyCount> getAppointmentDailyCount(String fromDate, String toDate, Location location,
```

Comment:
Wouldn't you want this query to exclude voided appointments?

Reply:
excluding voided appointments makes sense, updating this shortly